frontends/election-manager/src/screens/write_ins_transcription_screen.tsx

```diff
@@ -295,7 +295,7 @@ export function WriteInsTranscriptionScreen({
               <HorizontalRule color="#cccccc" />
               <p>
                 {showNewTranscriptionForm ? (
-                  <InlineForm>
+                  <InlineForm as="span">
```

Comment:
Does this have a UI impact or does it just fix the console warning I've been seeing?

Reply:
Thank you! This has been bugging me.